### PR TITLE
Next Step Template for Post Purchase

### PR DESCRIPTION
**Pull request by dmhenry**
Resolution of issue #234 - Next Step Template for Post Purchase
```
Post Purchase Test Extension (Build succeeded)

🔭 > If this is the first time you're testing a Post Purchase extension,
🔭 > please install the Chrome or Firefox browser extension from
🔭 > https://github.com/Shopify/post-purchase-devtools/releases.

🔭 > Once installed, simply enter your extension URL
🔭 > https://example.ngrok.io/extensions/00000000-0000-0000-0000-000000000002.
```

### CODE REVIEW

**Review by t6d — APPROVED** [review]
Looks good to me, but @vividviolet has the most context so I'd wait for her 👍

File: create/templates/checkout_post_purchase/next-steps.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+🔭 > If this is the first time you're testing a Post Purchase extension,
+🔭 > please install the Chrome or Firefox browser extension from
+🔭 > https://github.com/Shopify/post-purchase-devtools/releases.
+
+🔭 > Once installed, simply enter your extension URL
+🔭 > {{.IntegrationContext.PublicUrl}}/extensions/{{.Extension.UUID}}.
+
+🔭 > If you prefer to test your extension without using our browser extension,
```
Comment (vividviolet) [inline create/templates/checkout_post_purchase/next-steps.txt:9]: As we discussed, this seems to not work for Post Purchase so we should remove this. Can we also remove it from the [html page](https://github.com/Shopify/shopify-cli-extensions/blob/0dc47a6dfb024dd7e4900b7764aa5b67ff9c4b1e/api/root/templates/checkout_post_purchase/index.html.tpl#L38)?

File: api/api_test.go [hunk 2]
```diff
@@ -252,19 +252,6 @@ func TestAdminRedirect(t *testing.T) {
 	}
 }
 
-func TestPostPurchaseIndex(t *testing.T) {
```
Comment (vividviolet) [inline api/api_test.go:255]: Can we put back this test? It's still useful to assert that we get an html response for the Post Purchase index page. We don't need to check what's in the response anymore but we should still assert that it succeeds

Comment (dmhenry) [reply api/api_test.go:255]: That's fair.

**Pull request merged**